frontend/src/components/regions/CardDetailsModal/ReviewsTable.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import ReviewCard from "./ReviewsSbonelo";
+
+const useStyles = makeStyles({
+  root: {
+    minWidth: false
+  },
```

Comment:
please edit frontend/src/components/regions/CardDetailsModal/Reviews.js so it displays what we need. Dont make a whole new thing, rather just fix it in place.

If you have a car and you take it to the mechanic because the starter moter is broken, and then the mechanic doesn't fix anything and just gives you a new starter moter in your hand, would you be pleased?

Reply:
This has been edited and takes the review component from `frontend/src/components/regions/CardDetailsModal/Review.js`